specs/core/0_beacon-chain.md

```diff
@@ -562,7 +562,21 @@ If the user wishes to deposit more than 32 ETH, they would need to make multiple
 
 ### On startup
 
-Run the following code:
+A valid block with slot `0` (the "genesis block") has the following values. Other validity rules (eg. requiring a signature) do not apply.
```

Comment:
I think this is still a bit underspecified.
* The child of this block needs to not require the proposer signature
* Do/can committees at slot 0 attest to this block?

Need to make it clear at what slot (0 or 1) do the committee/proposer duties kick in

Reply:
I guess the resolution of this depends on whether or not we separate out the proposer to allow a minimum aggregation delay as we had discussed :)